addons/l10n_fr_pos_cert/models/account_bank_statement.py

```diff
@@ -13,7 +13,7 @@ def _unlink_except_created_by_pos(self):
         for statement in self:
             if not statement.company_id._is_accounting_unalterable() or not statement.journal_id.pos_payment_method_ids:
                 continue
-            if statement.state != 'open':
+            if any(line.state != 'draft' for line in statement.line_ids):
```

Comment:
Remove the condition. It's not relevant anymore

Reply:
hi @smetl ,
thanks for the review and feedback, if this is the case, the entire file can be dropped as there is no other codes other than this checking.

let me know your suggestion